README.md

```diff
@@ -100,5 +100,6 @@ is already know at:
 
 https://github.com/qxmpp-project/qxmpp/issues
 
+[support-chat]: https://i.kaidan.im/#qxmpp@muc.kaidan.im
```

Comment:
I don't want to link to a Kaidan site, that's why I haven't added this in the past.